curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/es6/explore-differences-between-the-var-and-let-keywords.portuguese.md

```diff
@@ -2,27 +2,59 @@
 id: 587d7b87367417b2b2512b3f
 title: Explore Differences Between the var and let Keywords
 challengeType: 1
-videoUrl: ''
```

Comment:
Please do not remove the videoUrl from the frontmatter.  This change should be reverted.